htlcswitch/decayedlog.go

```diff
@@ -64,16 +64,24 @@ type DecayedLog struct {
 // NewDecayedLog creates a new DecayedLog, which caches recently seen hash
 // shared secrets. Entries are evicted as their cltv expires using block epochs
 // from the given notifier.
-func NewDecayedLog(dbPath string,
+func NewDecayedLog(dbPath, dbFileName string, boltCfg *kvdb.BoltConfig,
```

Comment:
nice! didn't realize we were doing the other db's as well 👍

other possible dbs would be: macaroons.db, wtclient.db and watchtower.db, but those definitely aren't as high prio as channel.db so we can do them later. could also do wallet.db but that would require some changes to btcwallet unless we just go ahead and apply the compaction before it's opened...

Reply:
I think we should only do it for big DBs now. It does create more writes which for SSDs might not always be a good idea. Also, smaller DBs often tend to become a tad bigger after compaction. But it's very easy to turn on now, so we can always think about this again later.